tokio/src/net/unix/datagram/socket.rs

```diff
@@ -55,6 +170,29 @@ impl UnixDatagram {
     /// The runtime is usually set implicitly when this function is called
     /// from a future driven by a tokio runtime, otherwise runtime can be set
     /// explicitly with [`Handle::enter`](crate::runtime::Handle::enter) function.
```

Comment:
Also here on lines 48-49 (I can't select that far up). Would also be nice to capitalize "tokio" to "Tokio".